script.js

```diff
@@ -17,15 +17,18 @@ window.SmartBubbles = (function(win) {
   proto.handleEvent = function(evt) {
     switch(evt.type) {
       case 'animationend':
-        evt.currentTarget.classList.remove('smart-bubble-playing');
-        evt.currentTarget.classList.add('smart-bubble-stopped');
-        window.getComputedStyle(evt.currentTarget).animationName;
-        evt.currentTarget.classList.remove('smart-bubble-stopped');
-        window.getComputedStyle(evt.currentTarget).animationName;
-
         this.fireEvent('item-bubbled', ++this._finishedAnimationCount);
         if (this._targetAnimationCount === this._finishedAnimationCount) {
           this.playing = false;
+          [].forEach.call(this._elements, function(elem) {
+            elem.classList.remove('smart-bubble-playing');
+            elem.classList.add('smart-bubble-stopped');
+            // use getComputedStyle to force flushing re-style
```

Comment:
We should write more comment to say the original transition and animation may affect us.